<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PET_brg_ch_page</class>
 <widget class="QWidget" name="PET_brg_ch_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>1758</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="47" column="1">
      <widget class="QLabel" name="label_30">
       <property name="text">
        <string>Частота среза ФНЧ</string>
       </property>
      </widget>
     </item>
     <item row="55" column="0" colspan="3">
      <widget class="QLabel" name="label_34">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Параметры осциллографирования (DSO)</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLabel" name="label_53">
       <property name="text">
        <string>Общее число событий с переполнений кординатных сигналов</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLabel" name="label_47">
       <property name="text">
        <string>Общее число событий с опустошением кординатных сигналов</string>
       </property>
      </widget>
     </item>
     <item row="50" column="2">
      <widget class="QSpinBox" name="sb_qdc_predelay"/>
     </item>
     <item row="17" column="1">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>Номер канала регистрации событий</string>
       </property>
      </widget>
     </item>
     <item row="53" column="1">
      <widget class="QLabel" name="label_32">
       <property name="text">
        <string>Режим формирования метки времени</string>
       </property>
      </widget>
     </item>
     <item row="52" column="0" colspan="3">
      <widget class="QLabel" name="label_31">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Параметры формирования метки времени (TDC)</string>
       </property>
      </widget>
     </item>
     <item row="33" column="1">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Порог по уровню, мВ</string>
       </property>
      </widget>
     </item>
     <item row="40" column="2">
      <widget class="QDoubleSpinBox" name="dsb_blr_offset"/>
     </item>
     <item row="27" column="1">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Полярность координатных сигналов</string>
       </property>
      </widget>
     </item>
     <item row="54" column="1">
      <widget class="QLabel" name="label_33">
       <property name="text">
        <string>Поправка метки времени, пс</string>
       </property>
      </widget>
     </item>
     <item row="40" column="1">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Смещение, мВ</string>
       </property>
      </widget>
     </item>
     <item row="22" column="2">
      <widget class="QDoubleSpinBox" name="dsb_readout_temp_max"/>
     </item>
     <item row="25" column="2">
      <widget class="QDoubleSpinBox" name="dsb_adc_reference"/>
     </item>
     <item row="26" column="1">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Смещение сигнала ФЭУ, мВ</string>
       </property>
      </widget>
     </item>
     <item row="56" column="2">
      <widget class="QSpinBox" name="sb_dso_time_step"/>
     </item>
     <item row="36" column="0" colspan="3">
      <widget class="QLabel" name="label_28">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Параметры компенсации базовой линии (BLR)</string>
       </property>
      </widget>
     </item>
     <item row="26" column="2">
      <widget class="QDoubleSpinBox" name="dsb_in_offset"/>
     </item>
     <item row="28" column="2">
      <widget class="QComboBox" name="cb_in_scale"/>
     </item>
     <item row="50" column="1">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Предзадержка, нс</string>
       </property>
      </widget>
     </item>
     <item row="38" column="2">
      <widget class="QComboBox" name="cb_blr_fc"/>
     </item>
     <item row="47" column="2">
      <widget class="QComboBox" name="cb_fir_fc"/>
     </item>
     <item row="14" column="0" colspan="3">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Основные параметры</string>
       </property>
      </widget>
     </item>
     <item row="53" column="2">
      <widget class="QComboBox" name="cb_tdc_mode"/>
     </item>
     <item row="41" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Порог, мВ</string>
       </property>
      </widget>
     </item>
     <item row="49" column="2">
      <widget class="QComboBox" name="cb_qdc_mode"/>
     </item>
     <item row="56" column="1">
      <widget class="QLabel" name="label_35">
       <property name="text">
        <string>Шаг развертки по времени, нс/отcчет</string>
       </property>
      </widget>
     </item>
     <item row="31" column="1">
      <widget class="QLabel" name="label_27">
       <property name="text">
        <string>Режим работы</string>
       </property>
      </widget>
     </item>
     <item row="23" column="0" colspan="3">
      <widget class="QLabel" name="label_23">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Параметры АЦП и модуля сопряжения</string>
       </property>
      </widget>
     </item>
     <item row="57" column="1">
      <widget class="QLabel" name="label_36">
       <property name="text">
        <string>Шаг развертки по значению, мВ</string>
       </property>
      </widget>
     </item>
     <item row="31" column="2">
      <widget class="QComboBox" name="cb_cfd_mode"/>
     </item>
     <item row="58" column="2">
      <widget class="QSpinBox" name="sb_dso_prefetch"/>
     </item>
     <item row="19" column="1">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Напряжение смещения ФЭУ, В</string>
       </property>
      </widget>
     </item>
     <item row="51" column="2">
      <widget class="QDoubleSpinBox" name="dsb_qdc_scale"/>
     </item>
     <item row="39" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Длительность окна компенсации, нс</string>
       </property>
      </widget>
     </item>
     <item row="34" column="1">
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>Порог по приращению, мВ</string>
       </property>
      </widget>
     </item>
     <item row="41" column="2">
      <widget class="QDoubleSpinBox" name="dsb_blr_threshold"/>
     </item>
     <item row="39" column="2">
      <widget class="QSpinBox" name="sb_blr_gate_width"/>
     </item>
     <item row="35" column="1">
      <widget class="QLabel" name="label_38">
       <property name="text">
        <string>Соотношение</string>
       </property>
      </widget>
     </item>
     <item row="27" column="2">
      <widget class="QComboBox" name="cb_in_polarity"/>
     </item>
     <item row="33" column="2">
      <widget class="QDoubleSpinBox" name="dsb_cfd_l_threshold"/>
     </item>
     <item row="37" column="2">
      <widget class="QComboBox" name="cb_blr_mode"/>
     </item>
     <item row="29" column="2">
      <widget class="QDoubleSpinBox" name="dsb_ifmodule_temp_max"/>
     </item>
     <item row="28" column="1">
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>Масштабирующий коэффициент</string>
       </property>
      </widget>
     </item>
     <item row="46" column="2">
      <widget class="QComboBox" name="cb_fir_mode"/>
     </item>
     <item row="15" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Режим работы</string>
       </property>
      </widget>
     </item>
     <item row="16" column="2">
      <widget class="QSpinBox" name="sb_gate_width"/>
     </item>
     <item row="58" column="1">
      <widget class="QLabel" name="label_37">
       <property name="text">
        <string>Предвыборка, отсчетов</string>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="30" column="0" colspan="3">
      <widget class="QLabel" name="label_26">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Параметры дискриминатора (CFD)</string>
       </property>
      </widget>
     </item>
     <item row="38" column="1">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Частота среза ФНЧ</string>
       </property>
      </widget>
     </item>
     <item row="35" column="2">
      <widget class="QComboBox" name="cb_cfd_fract"/>
     </item>
     <item row="37" column="1">
      <widget class="QLabel" name="label_29">
       <property name="text">
        <string>Режим компенсации</string>
       </property>
      </widget>
     </item>
     <item row="32" column="2">
      <widget class="QComboBox" name="cb_cfd_lpf_mode"/>
     </item>
     <item row="21" column="2">
      <widget class="QSpinBox" name="sb_pm_bias_deviation"/>
     </item>
     <item row="48" column="0" colspan="3">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Параметры интегрирования (QDC)</string>
       </property>
      </widget>
     </item>
     <item row="57" column="2">
      <widget class="QDoubleSpinBox" name="dsb_dso_value_step"/>
     </item>
     <item row="54" column="2">
      <widget class="QSpinBox" name="sb_tdc_adj"/>
     </item>
     <item row="32" column="1">
      <widget class="QLabel" name="label_42">
       <property name="text">
        <string>Режим фильтрации</string>
       </property>
      </widget>
     </item>
     <item row="18" column="0" colspan="3">
      <widget class="QLabel" name="label_20">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Параметры ФЭУ и платы считывания</string>
       </property>
      </widget>
     </item>
     <item row="51" column="1">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Масштабирующий коэффициент</string>
       </property>
      </widget>
     </item>
     <item row="44" column="0" colspan="3">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Параметры фильтрации координатных сигналов (FIR)</string>
       </property>
      </widget>
     </item>
     <item row="21" column="1">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Максимально допустимое отклонение напряжения смещения ФЭУ, мВ</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QLabel" name="label_43">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Состояние</string>
       </property>
      </widget>
     </item>
     <item row="49" column="1">
      <widget class="QLabel" name="label_41">
       <property name="text">
        <string>Режим преобразования</string>
       </property>
      </widget>
     </item>
     <item row="15" column="2">
      <widget class="QComboBox" name="cb_mode"/>
     </item>
     <item row="16" column="1">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Длительность окна обработки, нс</string>
       </property>
      </widget>
     </item>
     <item row="46" column="1">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Режим фильтрации</string>
       </property>
      </widget>
     </item>
     <item row="29" column="1">
      <widget class="QLabel" name="label_25">
       <property name="text">
        <string>Максимально допустимая температура модуля сопряжения, °C</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_49">
       <property name="text">
        <string>Температура платы считывания, °C</string>
       </property>
      </widget>
     </item>
     <item row="25" column="1">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Опорное напряжение АЦП, мВ</string>
       </property>
      </widget>
     </item>
     <item row="22" column="1">
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>Максимально допустимая температура модуля считывания, °C</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_48">
       <property name="text">
        <string>Температура модуля сопряжения, °C</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="label_51">
       <property name="text">
        <string>Общее число событий с переполнением разрядной сетки АЦП</string>
       </property>
      </widget>
     </item>
     <item row="17" column="2">
      <widget class="QSpinBox" name="sb_event_src"/>
     </item>
     <item row="11" column="1">
      <widget class="QLabel" name="label_54">
       <property name="text">
        <string>Общее число событий с признаком наложения событий</string>
       </property>
      </widget>
     </item>
     <item row="34" column="2">
      <widget class="QDoubleSpinBox" name="dsb_cfd_d_threshold"/>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_50">
       <property name="text">
        <string>Состояние канала</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="label_52">
       <property name="text">
        <string>Загрузка канала регистрации, соб./цикл</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_55">
       <property name="text">
        <string>Номер цикла синхронизации</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="label_45">
       <property name="text">
        <string>Общее число событий с опустошением разрядной сетки АЦП</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLabel" name="label_46">
       <property name="text">
        <string>Общее число событий с признаком не корректности присвоенной метки времени</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="le_state"/>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="le_readout_temp"/>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="le_ifmodule_temp"/>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="le_sync_cycle"/>
     </item>
     <item row="6" column="2">
      <widget class="QLineEdit" name="le_utilisation"/>
     </item>
     <item row="7" column="2">
      <widget class="QLineEdit" name="le_adc_underflow"/>
     </item>
     <item row="8" column="2">
      <widget class="QLineEdit" name="le_adc_overflow"/>
     </item>
     <item row="9" column="2">
      <widget class="QLineEdit" name="le_qdc_underflow"/>
     </item>
     <item row="10" column="2">
      <widget class="QLineEdit" name="le_qdc_overflow"/>
     </item>
     <item row="11" column="2">
      <widget class="QLineEdit" name="le_gate_overlay"/>
     </item>
     <item row="12" column="2">
      <widget class="QLineEdit" name="le_ts_invalid"/>
     </item>
     <item row="19" column="2">
      <widget class="QDoubleSpinBox" name="dsb_pm_bias"/>
     </item>
     <item row="61" column="1">
      <widget class="QLabel" name="label_40">
       <property name="text">
        <string>Коэффициент передачи</string>
       </property>
      </widget>
     </item>
     <item row="60" column="0" colspan="3">
      <widget class="QLabel" name="label_39">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Дополнительные параметры</string>
       </property>
      </widget>
     </item>
     <item row="61" column="2">
      <widget class="QDoubleSpinBox" name="dsb_in_gain"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
